src/app_logic.js

```diff
@@ -0,0 +1,58 @@
+/*
+  eslint-disable no-unused-vars, no-alert, no-mixed-operators, import/prefer-default-export,
+  no-use-before-define
+*/
+
+import { getGif, getWeather, getDefaultWeather } from './api_handler';
+import { leftSide } from './left_side_dom';
+import { rigthSide } from './rigth_side_dom';
+import { handleForm } from './form_handler';
+
+const container = document.querySelector('.container');
+
+
+const loadDatas = async (city) => {
+  try {
+    const datas = await getWeather(city);
+    container.innerHTML = '';
+    const left = leftSide(datas);
+    container.appendChild(left);
+
+    const image = await getGif(datas.description);
+    const rigth = rigthSide(image);
+    container.appendChild(rigth);
+
+    listenSearch();
+  } catch (error) {
+    alert(error);
```

Comment:
Just alerting the actual error to the UI isn't really a good user experience, you do not expect your users to know how to read these errors